addOns/client/src/main/java/org/zaproxy/addon/client/ClientDetailsPanel.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
```

Comment:
2023

Reply:
It was started in 2022 .. but I've updated them all anyway 😉